src/test/java/htsjdk/samtools/reference/InMemoryReferenceSequenceFile.java

```diff
@@ -60,7 +61,14 @@ public ReferenceSequence getSequence(final String contig) {
     @Override
     public void close() throws IOException {
         map.clear();
-        index.clear();
+        if (index != null) {
```

Comment:
I guess that index cannot be null in this - the previous version was not checking it...